docs/WORLD_GENERATION.md

```
@@ -1,252 +1,72 @@
-# Generate world with specific seed
-pnpm dev --seed="test-seed-123"
-
-# Should produce identical district layout every time
-```
-
----
-
-*Every procedural element traces back to the master seed. No randomness escapes the chain.*
+  },
+}));
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)